bin/lambda-local

```diff
@@ -52,6 +52,8 @@
 		awsRequestId: _context.createInvokeId,
 		timeoutMs: timeoutMs
 	});
+    // export the LAMBDA_TASK_ROOT enviroment variable
+    process.env['LAMBDA_TASK_ROOT'] = process.cwd();
```

Comment:
As far as I know, LAMBDA_TASK_ROOT is the directory where the zip is exracted. But ```__dirname``` would give me the  directory, where the lambda_local Script is, which is not what it would be when running on lambda.